### DISCUSSION TITLE
Facing issue with Vercel deployment Error: The Edge Function "src/middleware" is referencing unsupported modules:	- index.js This error comes when upgrading to the next version 13.4.3. I tried to make a build in our local server then not facing the issue.

### DISCUSSION THREAD

**Discussion by sedar-shahid** [vercel/next.js#50887] [Help]
### Summary

Error: The Edge Function "src/middleware" is referencing unsupported modules:- index.js: ./resvg.wasm?module, ./yoga.wasm?module
NOW_SANDBOX_WORKER_EDGE_FUNCTION_UNSUPPORTED_MODULES: The Edge Function "src/middleware" is referencing unsupported modules:- index.js: ./resvg.wasm?module, ./yoga.wasm?module

**Comment by icybergenome** [comment]
@sedar-shahid any clue what went wrong? We are facing the same issue
